shakenfist/external_api/app.py

```diff
@@ -635,10 +635,14 @@ def post(self, name=None, cpus=None, memory=None, network=None,
         if not namespace:
             namespace = get_jwt_identity()
 
+        # Only system can specify a uuid
+        if instance_uuid and get_jwt_identity() != 'system':
+            return error(401, 'Only system can specify an instance uuid')
+
         # If accessing a foreign namespace, we need to be an admin
         if get_jwt_identity() not in [namespace, 'system']:
             return error(401,
-                         'only admins can create resources in a different namespace')
+                         'Only admins can create resources in a different namespace')
```

Comment:
Currently errors start with lower case. Are we changing to upper case?

Reply:
Fixed.